composer/lib/dependabot/composer/update_checker/version_resolver.rb

```diff
@@ -471,6 +479,11 @@ def path_dependency_files
             dependency_files.select { |f| f.name.end_with?("/composer.json") }
         end
 
+        def zipped_path_dependency_files
+          @zipped_path_dependency_files ||=
+            dependency_files.select { |f| f.name.end_with?(".zip", ".gitkeep") }
```

Comment:
Should we make these constants? Ditto for .gitkeep and others

Reply:
I don't think that would add much for safety, and would detract from readability depending on how that is done.